src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/OrderedHashtable.cs

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I wonder if this file should be in `src/System.Management.Automation/engine`, since we will likely want to make this a first-class PowerShell type at some point?

Reply:
This was discussed in the Cmdlet WG and agreed it should be in SMA, will move